LayoutTests/ipc/send-gpu-GetShareableBitmap-RemoteRenderingBackend.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html><!-- webkit-test-runner [ IPCTestingAPIEnabled=true ] -->
+<script src="../resources/js-test.js"></script>
+<script>
+function fuzz() {
+    description("Send IPC messages between GPU process and WebProcess to create RenderingBackend then ImageBuffer then get the ShareableBitmap");
+    window.jsTestIsAsync = true;
+
+    if (window.testRunner) {
+        testRunner.waitUntilDone();
+        testRunner.dumpAsText();
+        finishJSTest();
+    }
```

Comment:
You do not need this. `window.jsTestIsAsync = true;` is exactly the same as `testRunner.waitUntilDone();`. Including `js-test.js` will call `testRunner.dumpAsText();`. Also you do not need to check `window.testRunner`.